Sources/PodToBUILD/ObjcLibrary.swift

```diff
@@ -690,7 +690,7 @@ private func extractResources(patterns: [String]) -> [String] {
 
 private func extractHeaders(patterns: [String]) -> [String] {
     return patterns.flatMap { (p: String) -> [String] in
-        pattern(fromPattern: p, includingFileTypes: ["h"])
+        pattern(fromPattern: p, includingFileTypes: ["h", "hpp"])
```

Comment:
While we are at it - can you add `hxx`, which is another source file type people use for C++ headers.